docs/OV_Runtime_UG/multi_device.md

```diff
@@ -1,39 +1,41 @@
 # Running on multiple devices simultaneously {#openvino_docs_OV_UG_Running_on_multiple_devices}
 
-## Introducing the Multi-Device Plugin (C++)
 
-@sphinxdirective
-.. raw:: html
 
-    <div id="switcher-cpp" class="switcher-anchor">C++</div>
-@endsphinxdirective
+The Multi-Device execution mode, or MULTI for short, uses a "virtual" or a "proxy" device, which does not bind to a specific type of hardware. Instead, it assigns available computing devices to particular inference requests, which are then executed in parallel. 
 
-The Multi-Device plugin automatically assigns inference requests to available computational devices to execute the requests in parallel. By contrast, the Heterogeneous plugin can run different layers on different devices but not in parallel. The potential gains with the Multi-Device plugin are:
+The potential gains from using Multi-Device execution are:
+* improved throughput from using multiple devices at once,
+* increase in performance stability due to multiple devices sharing inference workload.
 
-* Improved throughput from using multiple devices (compared to single-device execution)
-* More consistent performance, since the devices share the inference burden (if one device is too busy, another can take more of the load)
+What is important, the Multi-Device mode does not change tha application logic, so it does not require you to explicitly compile the model on every device or create and balance inference requests. From the application point of view, this is just another plugin that handles the actual machinery.
```

Comment:
"just another plugin that handles " - > "just a conventional **device** that **internally** handles the actual machinery"

Reply:
sounds better and more formal, gladly changing... done.